packages/lib/src/components/Dropin/components/DropinComponent.tsx

```diff
@@ -157,7 +188,29 @@ export class DropinComponent extends Component<DropinComponentProps, DropinCompo
                     <div className={`adyen-checkout__dropin adyen-checkout__dropin--${status.type}`}>
                         {isRedirecting && status.props.component && status.props.component.render()}
                         {isLoading && status.props && status.props.component && status.props.component.render()}
-                        {!!hasPaymentMethodsToBeDisplayed && (
+
+                        {showFastlane && (
```

Comment:
Do we need this extra `showFastlane` variable? Isn't it the same as `!showDefaultPaymentMethodList`?